envs/doom/multiplayer/doom_multiagent.py

```diff
@@ -8,6 +8,12 @@
 DEFAULT_UDP_PORT = int(os.environ.get('DOOM_DEFAULT_UDP_PORT', 40300))
 log.info('Default UDP port is %r', DEFAULT_UDP_PORT)
 
+# This try except block is increase the env timeout connection flag in travis
+try:
+    vizdoom_env_timeout = int(os.environ['TRAVIS_VIZDOOM_ENV_TIMEOUT'])
```

Comment:
Since you moved this to the global scope, it's better to follow the convention and use the FULL_CAPS name for it. But it's okay. Actually I like it in the global scope